content/sensu-go/5.10/guides/reduce-alert-fatigue.md

```diff
@@ -61,21 +65,142 @@ sensuctl handler update slack
 Follow the prompts to add the `hourly` and `is_incident` filters to the Slack
 handler.
 
+#### Using a Filter Asset
+
+If you're not already familiar with [assets][asset-reference], take a minute or two and read over our [guide to installing plugins with assets][asset-guide]. This will help you understand what an asset is and how they are used in Sensu. 
+
+The first step we'll need to take is to obtain a filter asset that will allow us to replicate the behavior we used when we created the `hourly` filter via `sensuctl`. Let's use the [fatigue check asset][fatigue-check-asset] from the [Bonsai Asset Index][bonsai-io]. You can download the asset directly by running the following:
+
+{{< highlight shell >}}
+curl -s https://bonsai.sensu.io/release_assets/nixwiz/sensu-go-fatigue-check-filter/0.1.3/any/noarch/download -o sensu-fatigue-check-filter-asset.yml
+{{< /highlight >}}
+
+That should give us a file that looks like this:
+
+{{< highlight yaml >}}
+---
+type: Asset
+api_version: core/v2
+metadata:
+  name: fatigue_check
+  namespace: default
+  labels: {}
+  annotations: {}
+spec:
+  url: https://github.com/nixwiz/sensu-go-fatigue-check-filter/releases/download/0.1.3/sensu-go-fatigue-check-filter_0.1.3.tar.gz
+  sha512: b58e7736fdb77901c243eac10a3c147f5cb4d6ef70966764b8735396bf207153ce8cd52afa0ddd5c6f7602949b2bfc76c0c28bb7843b86c265545ae770c70346
+  filters: []
+{{< /highlight >}}
+
+Once we have created our file for registering an asset, we'll need to apply it. To apply it, run the following command:
+
+{{< highlight shell >}}
+sensuctl create -f sensu-fatigue-check-filter-asset.yml
+{{< /highlight >}}
+
+Excellent! You've registered the asset. We still need to create our filter. We'll use the following configuration for creating the actual filter. In this case, we'll call it `sensu-fatigue-check-filter.yml`:
+
+{{< highlight yaml >}}
+---
+type: EventFilter
+api_version: core/v2
+metadata:
+  name: fatigue_check
+  namespace: default
+spec:
+  action: allow
+  expressions:
+  - fatigue_check(event)
+  runtime_assets:
+  - fatigue-check-filter
+{{< /highlight >}}
+
+And we'll go ahead and create it:
+
+{{< highlight shell >}}
+sensuctl create -f sensu-fatigue-check-filter.yml
+{{< /highlight >}}
+
+Now that we've created the filter asset and the filter, let's move on to the check annotations needed for the asset to work properly. 
+
+#### Annotating a check for filter asset use
+
+Now that we've created the filter, we'll need to make some additions to any checks we want to use the filter with. Let's look at an example CPU check:
+
+{{< highlight yaml >}}
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: linux-cpu-check
+  namespace: default
+  annotations:
+    fatigue_check/occurrences: '1'
+    fatigue_check/interval: '3600'
+    fatigue_check/allow_resolution: 'false'
```

Comment:
I'd recommend adding some explicit description about what these annotations are doing and what exactly the behavior of the resulting filter will be.